### ISSUE TITLE
[REQ] fix GH license

### ISSUE THREAD

**Issue by forart** [bbc/audio-offset-finder#44]
Hi there, 1st of all thanks for your cool work !

As you know, showing licensing terms "at glance" is extremely important to let anyone better/faster understand how to reuse/adapt/modify sources (and not only) in other open projects and vice-versa.

Although it may sounds like a minor aspect, GH's license file "uncompliance" causes an inconsistent generation of the relative badge too:

![](https://flat.badgen.net/github/license/bbc/audio-offset-finder/?label=LICENSE)
(badge-generator URL: https://flat.badgen.net/github/license/bbc/audio-offset-finder/?label=LICENSE)

You can easily set a standardized one through the GH's license wizard tool.

Then, since we've listed the project under the [AUDIO category](https://github.com/forart/HyMPS#-) \ [Tools section](https://github.com/forart/HyMPS/blob/main/A_Tools.md#--) \ [Alignment/synch subsection](https://github.com/forart/HyMPS/blob/main/A_Tools.md#alignmentsynch-) , we wondered if it's the right placing for it or there's a better one.

Last but not least, we suggest you to keep in consideration both [AudioAlign](https://github.com/protyposis/AudioAlign) by @protyposis - even if it'sunfortunally adbandoned/not updated - and [Audalign](https://github.com/benfmiller/audalign) by @benfmiller, that are the most interesting similar projects on the scene.

Thanks in advance.

**Comment by chrisn** [comment]
What change is needed? We have https://github.com/bbc/audio-offset-finder/blob/master/COPYING

**Comment by forart** [comment]
1st of all thanks for the fast reply.

This is a "GH-compliant" Apache 2.0 license...
https://github.com/soundhive/cpp-fingerprint-generator/blob/master/LICENSE

...than you'll achieve this in the repo-page:
![apache](https://github.com/bbc/audio-offset-finder/assets/171307/a07e2469-450d-4ecd-b3e3-ebb4ac4bec3d)

Thanks again.

**Comment by chrisn** [comment]
Thanks! So I guess the issue is that GitHub doesn't recognise our license file because it's an edited version of the Apache 2.0 license (it adds copyright and author information at the top).

**Comment by stephenjolly** [comment]
Possibly also that the filename is not as expected?  We could replace the license information in COPYING with a pointer to a separate LICENSE file and update README.md accordingly.

**Comment by forart** [comment]
> Thanks! So I guess the issue is that GitHub doesn't recognise our license file because it's an edited version of the Apache 2.0 license (it adds copyright and author information at the top).
>
Yes, as the template states, the "copyright and author information" must be specified at row 189:
![license](https://github.com/bbc/audio-offset-finder/assets/171307/57bc4510-bf6d-4259-aa75-0d1158ceecbf)

Hope that helps

**Comment by chrisn** [comment]
The license badge looks to be ok now: https://flat.badgen.net/github/license/bbc/audio-offset-finder?label=LICENSE

Thank you @forart!

**Comment by stephenjolly** [comment]
I've also added @forart's links to similar projects to the README - many thanks for those, too.
